low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -150,7 +192,8 @@ def get_realstor_encl_data(fru: str):
         return fru_data
 
 
-# if __name__ == "__main__":
-#     storage = StorageMap()
-#     health_data = storage.get_health_info(rpath="nodes[0]>storage[0]>hw>controllers")
-#     print(health_data)
+if __name__ == "__main__":
+    storage = StorageMap()
+    health_data = storage.get_health_info(rpath="nodes[0]>storage[0]>hw>platform_sensors")
+    import pprint
+    pprint.pprint(health_data)
```

Comment:
These lines should be commented.
This module can be executed by any script in command line by any chance. So __main__ will be executed.

Reply:
Done.